apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -485,10 +485,10 @@ export class SubscribersController {
       organizationId: user.organizationId,
       environmentId: user.environmentId,
       subscriberId: subscriberId,
-      page: query.page != null ? parseInt(query.page) : 0,
+      page: query.page != null ? parseInt(query.page as any) : 0,
```

Comment:
are we certain that PaginationRequestDto paras are validated in the transport dto phase, if we do then I think it should be saved to remove the parsing because if it is not number @IsInt() will throw an exception.